tests/LoggerManager.spec.ts

```diff
@@ -0,0 +1,150 @@
+import { Client, ClientEvents, Intents } from 'discord.js';
+
+import { LoggerManager, IModule } from '../src';
+import { generateTestModules, getTestEvent } from './_global';
+
+describe('LoggerManager', () => {
+	let modules: Partial<IModule>[];
+	let client: Client;
+
+	beforeEach(() => {
+		client = new Client({ intents: [Intents.FLAGS.GUILDS] });
+		client.removeAllListeners();
+
+		modules = [
+			...generateTestModules()
+		];
+	});
+
+	it('should instanciate correctly', () => {
+		expect(() => new LoggerManager(client, modules)).not.toBeNull();
+	});
+
+	it('should not throw error when "modules" is not defined', () => {
+		expect(() => new LoggerManager(client)).not.toBeNull();
+	});
+
+	describe('listenWithAllModulesTo', () => {
+		it('should add event to the events list', () => {
+			const sut = new LoggerManager(client, modules);
+			const event = getTestEvent();
+
+			expect(client.listenerCount(event)).toBe(0);
+			sut.listenWithAllModulesTo(event);
+			expect(client.listenerCount(event)).toBe(modules.length);
+		});
+
+		it('should not add twice an event to the events list', () => {
```

Comment:
This test description should be "should not add twice the modules listeners to this event"